api-builder.js

```diff
@@ -1,15 +1,18 @@
 module.exports = buildApi
 
 var routes =
-  { get: require('./endpoints/get')
-  , post: require('./endpoints/post')
-  , put: require('./endpoints/put')
-  , patch: require('./endpoints/patch')
-  , 'delete': require('./endpoints/delete')
-  }
+      { get: require('./endpoints/get')
+      , post: require('./endpoints/post')
+      , put: require('./endpoints/put')
+      , patch: require('./endpoints/patch')
+      , 'delete': require('./endpoints/delete')
+      }
+  , EventEmitter = require('events').EventEmitter
 
 function buildApi(service, urlRoot, router, logger, middleware, verbs) {
 
+  var apiBuilder = new EventEmitter()
```

Comment:
I also hate this aspect of the revealing module pattern. I've never bothered fixing it before, but I think it is worth doing it as @bengourley suggests.